### ISSUE TITLE
fixed FFT length?

### ISSUE THREAD

**Issue by JBloodless** [Qinwen-Hu/dparn#10]
Does your model expect only spectrum with 601 freq bins? So I can't use, for example, nfft=256, only nfft=1200?

**Comment by Qinwen-Hu** [comment]
Yes, the FFT length is fixed if you want to use the provided code directly, since 'SpecCompress.npy' is a fixed matrix. But you can generate a different version of 'SpecCompress.npy' if you want to use different FFT length, with the code I attached here
[scm.txt](https://github.com/Qinwen-Hu/dparn/files/9439055/scm.txt)
.
